tests/lib/k8s/test_builders.py

```diff
@@ -176,17 +176,50 @@ def test_get_anti_affinity_zone_enabled_for_process(self, parse_yaml_mock):
 
     def test_get_image_name_branch(self):
         """Returns the URL of the docker image if tag and branch are provided"""
-        app_config = {"app": "app", "registry": {"organization": "my-organization"}}
+        app_config = {
+            "app": "app",
+            "registry": {"platform": "docker-hub", "id": "id-1"},
+            "docker": {
+                "registries": {
+                    "docker-hub": [
+                        {"id": "id-1", "organization": "my-organization-1"},
+                        {"id": "id-2", "organization": "my-organization-2"},
+                    ],
+                },
+            },
+        }
```

Comment:
I would personally not have this over-complicated
There is no real reason to have several registries ATM